src/pages/app-about/app-about.scss

```diff
@@ -124,6 +124,18 @@
       max-width: 420px;
       text-align: left;
     }
+
+    button {
+      &:hover {
+        background: #ff5d00;
+        border-color: white;
+      }
+
+      &:not(:disabled):active {
+        background-color: #ff5d00;
```

Comment:
We should use the global variable for this hexcode to keep it consistent.

Reply:
you are right, fixed